src/main/java/com/dabsquared/gitlabjenkins/publisher/GitLabMessagePublisher.java

```diff
@@ -35,7 +36,6 @@
     private String abortNoteText;
     private String unstableNoteText;
 
-    @DataBoundConstructor
     public GitLabMessagePublisher(boolean onlyForFailure, boolean replaceSuccessNote, boolean replaceFailureNote, boolean replaceAbortNote, boolean replaceUnstableNote,
```

Comment:
Probably best to deprecate the constructor.

Reply:
Done.